requests/test/src/requests/RequestTests.scala

```diff
@@ -268,5 +294,86 @@ object RequestTests extends TestSuite{
       assert(response.headers.keySet.map(_.toLowerCase).contains("content-length"))
       assert(response.headers.keySet.map(_.toLowerCase).contains("content-type"))
     }
+
+    /**
+      * Compress with each compression mode and call server. Server expands
+      * and passes it back so we can compare
+      */
+  test("compressionData") {
+    import Compress._
+    val str = "I am deflater mouse"
+    Seq(None, Gzip, Deflate).foreach(c => 
```

Comment:
Formatting is weird here

Reply:
Maybe you could be slightly more specific on what is weird where you don't like the formatting. One of then looks a little over-indented, but I wouldn't know what to change on the others.  I've never done this before, and in my imagination you were going to merge in the parts you found agreeable and hit format;-)